docs/tutorials/00-landscaper-installation.md

```diff
@@ -0,0 +1,138 @@
+# Install the Landscaper
+
+This tutorial describes how to install the Landscaper on a Kubernetes cluster and optionally how to install an OCI registry
+on a Gardener Shoot cluster.
+
+### Prerequisites
+
+- Kubernetes cluster
```

Comment:
isnt' it also required for the cluster to be a shoot when the registry is installed?
Or alternativly a ingress and a certificate management:

But I think we should mention that

Reply:
Yes, might be a bit confusing.
I have moved this prerequisite to alternative 2.
Alternative 1 requires a gardener shoot cluster with nginx ingress installed.
Alternative 2 expects an external OCI registry, therefore no ingress or cert manager is required.